src/visualization/include/iDynTree/MeshcatVisualizer.h

```diff
@@ -58,11 +58,22 @@ namespace iDynTree
                        const iDynTree::Span<const double> &jointPositions,
                        const std::string &modelName);
 
-    bool setSphere(const std::string &name,
-                   const iDynTree::Position &position,
-                   const double radius,
-                   const iDynTree::VectorFixSize<3> &color);
+    /**
+     * Load a mesh in the visualizer.
+     * @param name the name of the sphere. Each geometry you add needs to have an unique name.
+     * @param radius radius of the sphere.
+     * @param color the color of the mesh.
+     * @return True in case of success false otherwise.
+     */
+    bool loadSphere(const double radius,
+                    const iDynTree::Span<const double> &color,
+                    const std::string &name);
+
+    bool setPrimitiveGeometryTransform(const iDynTree::Transform &world_T_geometry,
+                                       const std::string &geometryName);
 
+    bool setPrimitiveGeometryTransform(const iDynTree::MatrixView<const double> &world_T_geometry,
+                                       const std::string &geometryName);
```

Comment:
Can you add the documentation?

Reply:
It is available with the last commit 09cdd73c0e9c5e4409bc5564110a3ba38f9d8acb. I added 1 comment for both implementations. Do I have to duplicate it?

Reply:
I think yes since they are rendered by doxygen to generate the documentation

Reply:
I think yes since they are rendered by doxygen to generate the documentation

Reply:
There is also a way for Doxygen to automatically insert the doc of another method, but for sure the easiest fix is to duplicate.

Reply:
With this commit 798df73f582b933c72e3171bb3da3c8c1adbcefa everything should be settled. Let me know if any other edit is required @GiulioRomualdi @traversaro.